tests/test_crud/test_build.py

```diff
@@ -0,0 +1,18 @@
+from alws.crud import build
+
+from tests.mock_classes import BaseAsyncTestCase
+
+
+class TestBuildCrud(BaseAsyncTestCase):
+
+    async def test_build_delete(
+        self,
+        session,
+        base_platform,
+        base_product,
+        create_errata,
+        build_done,
+        build_for_release,
+    ):
+        await session.close()
+        await build.remove_build_job(session, build_for_release.id)
```

Comment:
I think it's better to add a test for the `/{build_id}/remove/` endpoint instead, because we can cover the endpoint with a test too, rather than only that function

Reply:
Fixed